homeassistant/components/zwave/__init__.py

```diff
@@ -778,49 +969,18 @@ def __init__(self, value, domain):
 
     def network_value_changed(self, value):
         """Called when a value has changed on the network."""
-        if self._value.value_id == value.value_id:
-            return self.value_changed()
-
-        dependent_ids = self._get_dependent_value_ids()
-        if dependent_ids is None and self._value.node == value.node:
-            return self.value_changed()
-        if dependent_ids is not None and value.value_id in dependent_ids:
+        if value.value_id in [v.value_id for v in self.values if v]:
             return self.value_changed()
 
     def value_changed(self):
         """Called when a value for this entity's node has changed."""
-        if not self._value.node.is_ready:
-            self._update_ids()
         self._update_attributes()
         self.update_properties()
         # If value changed after device was created but before setup_platform
         # was called - skip updating state.
         if self.hass and not self._update_scheduled:
             self.hass.add_job(self._schedule_update)
 
-    def _update_ids(self):
-        """Update value_ids from which to pull attributes."""
-        if self._wakeup_value_id is None:
-            self._wakeup_value_id = self.get_value(
-                class_id=const.COMMAND_CLASS_WAKE_UP, member='value_id')
-        if self._battery_value_id is None:
-            self._battery_value_id = self.get_value(
-                class_id=const.COMMAND_CLASS_BATTERY, member='value_id')
-        if self._power_value_id is None:
-            self._power_value_id = self.get_value(
-                class_id=[const.COMMAND_CLASS_SENSOR_MULTILEVEL,
-                          const.COMMAND_CLASS_METER],
-                label=['Power'], member='value_id',
-                instance=self._value.instance)
```

Comment:
Yeah, we can always refine this later, but this at least gives us a jumping off point.

Thanks again for the great feedback!